decentraland/screens/BuyLandScreen.js

```diff
@@ -45,11 +45,13 @@ export class BuyLandScreen extends React.Component {
     const {
       navigation,
       accountInfo,
+      myAssets,
       removeLandForSale,
       prependLandForSaleToList,
       removeMyAssetFromList,
       addToMyAssetsList,
     } = props;
+    let { removeMyAssetFromList, addToMyAssetsList } = props;
```

Comment:
Did this need to be a `let` as opposed to a `const`?

Reply:
I've thought that will be necessary to override these functions but it didn't. Just fixed.

Reply:
Okay great, thanks. Resolving this.